src/db/table/payment_type.sql

```diff
@@ -0,0 +1,8 @@
+-- Payment types represent the supported payment methods
+CREATE TABLE `payment_type` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `uq_name` varchar(25) NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `idx_payment_type` (`uq_name`)
```

Comment:
Index names should be `idx_<column name>`, so in this case, `idx_name`. Keys are unique to the table only. Please fix for this and all other occurrences.